src/routes/webhooks/paddle.ts

```diff
@@ -36,8 +36,8 @@ const updateUserSubscription = async ({
     return false;
   }
   const subscriptionType = data.data?.items.reduce((acc, item) => {
-    if (item.price?.id && planTypes()[item.price.id]) {
-      acc = planTypes()[item.price.id];
+    if (item.price?.id && remoteConfig.vars?.pricingIds?.[item.price.id]) {
+      acc = remoteConfig.vars.pricingIds?.[item.price.id] ?? '';
```

Comment:
This was just TS complaining

Reply:
its meant to complain yes 😆 so we can fallback by design

Reply:
Could define `remoteConfig.vars?.pricingIds?` as a constant in this parent function

Reply:
`const pricingIds = remoteConfig.vars?.pricingIds || []` maybe